Aspire Budgeting/Facilities/GoogleSheetsManager.swift

```diff
@@ -253,6 +255,19 @@ final class GoogleSheetsManager: ObservableObject {
     }
   }
   
+  func fetchAccountBalances(spreadsheet: File) {
+    os_log("Fetching Account Balances",
+           log: .sheetsManager, type: .default)
+    
+    let range = "Dashboard!B10:C"
+    fetchData(spreadsheet: spreadsheet, spreadsheetRange: range) { (valueRange) in
+      if let values = valueRange.values as? [[String]] {
+        self.accountBalancesMetadata = AccountBalancesMetadata(metadata: values)
+      }
+    }
+    
+  }
+  
   private func createSheetsValueRangeFrom(amount: String,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/9daa0d03405a5508dbc294588fce3ed145122088/Aspire Budgeting/Facilities/GoogleSheetsManager.swift#L271"></span>Function should have 5 parameters or less: it currently has 6<br />
<code>function_parameter_count</code> <code>GoogleSheetsManager.swift:271</code></td>
    </tr>
  </tbody>
</table>